Update the custom medication dictionary

iaindillingham opened the pull request:
Thanks for providing the mappings, @chrisjwood16. For background, see [1]; and opensafely-core/ehrql#1270.

[1]: https://bennettoxford.slack.com/archives/C03HVM72273/p1687786350885009

inglesp reviewed (approved):
Is it worth having another column for the drug name?

iaindillingham commented:
> Is it worth having another column for the drug name?

Do you mean to the CSV and the table? Or just to the CSV? I've added it to the CSV, and updated the tests, because I wasn't sure that adding an extra column to the CSV wouldn't break the `INSERT`. (It doesn't!) However, I haven't added it to the table. (Possibly this should have featured in opensafely-core/ehrql#1348?)

inglesp commented:
> Do you mean to the CSV and the table? Or just to the CSV?

Just the CSV!
